### PR TITLE
Re-disable GPU, but only on windows

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: R/chromote.R [hunk 1]
```diff
@@ -599,23 +599,26 @@ is_missing_linux_user <- cache_value(function() {
...
+#'   * Disables GPU hardware acceleration. If software renderer is not in place, then the GPU process won't launch.
 #' * [`"--no-sandbox"`](https://peter.sh/experiments/chromium-command-line-switches/#no-sandbox)
 #'   * Only added when `CI` system environment variable is set, when the
 #'     user on a Linux system is not set, or when executing inside a Docker container.
-#'   * \verb{Disables the sandbox for all process types that are normally sandboxed. Meant to be used as a browser-level switch for testing purposes only}
+#'   * Disables the sandbox for all process types that are normally sandboxed. Meant to be used as a browser-level switch for testing purposes only
 #' * [`"--disable-dev-shm-usage"`](https://peter.sh/experiments/chromium-command-line-switches/#disable-dev-shm-usage)
 #'   * Only added when `CI` system environment variable is set or when inside a docker instance.
-#'   * \verb{The /dev/shm partition is too small in certain VM environments, causing Chrome to fail or crash}
+#'   * The `/dev/shm` partition is too small in certain VM environments, causing Chrome to fail or crash.
```
Comment (hadley) [inline R/chromote.R:611]: Not sure why these entire sentences were wrapped in `\verb`

**Comment by wch** [comment]
Why is this change needed?

**Comment by hadley** [comment]
So the CI checks don't hang forever

**Comment by dmurdoch** [comment]
Is it possible for `chromote` users to control this choice?  I've heard complaints about the speed of snapshots in `rgl` for a while now.  It uses `webshot2::webshot()`, which uses `chromote`.  There are so many layers that I haven't had a chance to try to figure out why it is so slow, but an emulated GPU could be an explanation.

**Comment by hadley** [comment]
Maaaaybe, but given that it causes it to hang on GHA, I don't have a lot of confidence that it will generally help.

**Pull request merged**